projects/valtimo/access-control/src/lib/services/permission-api.service.ts

```diff
@@ -0,0 +1,43 @@
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {ConfigService} from '@valtimo/config';
+import {map, Observable, tap} from 'rxjs';
+import {
+  PermissionRequest,
+  PermissionRequestCollection,
+  PermissionResponse,
+  ResolvedPermissions,
+} from '../models';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PermissionApiService {
+  private valtimoEndpointUri: string;
+
+  constructor(private readonly http: HttpClient, private readonly configService: ConfigService) {
+    this.valtimoEndpointUri = `${this.configService.config.valtimoApi.endpointUri}v1/permissions`;
```

Comment:
Works for now, but what if we in the future support both v1 and v2 endpoints in this service? So I guess either call it something like permissionsEndpointV1Uri or just use valtimoEndpointUri like in other services.

Maybe not for this PR, but I'm thinking there's a lot of code duplication with setting valtimoEndpointUri each time in each api service. We could just make an InjectionToken once for the api uri and use that